libs/search-widget/public/i18n/en.json

```diff
@@ -1,5 +1,6 @@
 {
 	"answer.chat-action": "Ask more",
+	"answer.download-log": "Download log",
```

Comment:
shouldn't this be plural (logs)? (if so you need to update all the translations)

Reply:
it is just the last log entry